### PR TITLE
Layout / Container

### PR DESCRIPTION
**Pull request by chris-pearce**
http://codepen.io/team/westfieldlabs/full/GiCzg/

### CODE REVIEW

File: layout/_l-container.scss [hunk 1]
```diff
@@ -15,28 +25,20 @@
...
-    margin: 0 auto;
-  }
-}
+/**
+ * Modifier: full bleed.
+ */
 
-/* Modifier: wide container */
-.l-container--wide {
+.l-container--full-bleed {
```
Comment (kllevin) [inline layout/_l-container.scss:33]: Way better than "wide" :+1:

Comment (chris-pearce) [inline layout/_l-container.scss:33]: Yeah trying to be a lot more descriptive with names.

**Comment by AlecRaeside** [comment]
LGTM

File: core/mixins/_media-queries.scss [hunk 2]
```diff
@@ -202,28 +202,21 @@
...
 	@if $respond-range == lap {
-		@media (min-width: ($lap-start / 16)+em) and (max-width: ($lap-end / 16)+em) {
-			@content;
-		}
-	}
-
-	// Lap small
-	@if $respond-range == lap-sml {
-		@media (min-width: ($lap-start / 16)+em) and (max-width: ($lap-sml-end / 16)+em) {
+		@media (min-width: (641 / 16)+em) and (max-width: (1365 / 16)+em) {
```
Comment (kllevin) [inline core/mixins/_media-queries.scss:205]: you can't use $lap, etc. here?

File: core/settings/_toggles.scss [hunk 3]
```diff
@@ -75,51 +73,25 @@ $viewport-device-width: true !default;
  * Apply classes at specific breakpoints.
  */
 
-// Width classes mainly used with the grid
-$width-classes-at-breakpoint-palm-large:              false !default;
-// N.B. we turn `non-palm` on by default as the grid needs this
-$width-classes-at-breakpoint-non-palm:                true !default;
```
Comment (kllevin) [inline core/settings/_toggles.scss:81]: Are you getting rid of all non-palm?

File: core/mixins/_media-queries.scss (hunk 2 above)
Comment (chris-pearce) [inline core/mixins/_media-queries.scss:205]: This ain't ready for review :) right now I'm hacking things together so I can get the grid done, I'll post an update in HC when the grid is ready for review.

File: core/settings/_toggles.scss (hunk 3 above)
Comment (chris-pearce) [inline core/settings/_toggles.scss:81]: Yep, just use `$lap` instead but like mentioned above this is being hacked together ATM so I can complete the grid.

**Pull request merged**